yarn.lock

```diff
@@ -7062,6 +7173,18 @@ sucrase@^3.12.1:
     pirates "^4.0.1"
     ts-interface-checker "^0.1.9"
 
+sucrase@^3.20.0:
+  version "3.20.2"
+  resolved "https://registry.yarnpkg.com/sucrase/-/sucrase-3.20.2.tgz#28a28dc58a55be0d6916d5c9b2440d203e9ffe62"
+  integrity sha512-EdJ5M6VEvToIZwIWiZ71cxe4CklDRG8PdSjUSst+BZCUGlaEhnrdQo/LOXsuq3MjWRbfepg1XTffClK0Tmo0HQ==
+  dependencies:
+    commander "^4.0.0"
+    glob "7.1.6"
+    lines-and-columns "^1.1.6"
+    mz "^2.7.0"
+    pirates "^4.0.1"
+    ts-interface-checker "^0.1.9"
+
```

Comment:
Please delete both `sucrase` blocks & re-run yarn to consolidate this dependency.